AFQ/api/group.py

```diff
@@ -886,6 +886,15 @@ def assemble_AFQ_browser(self, output_path=None, metadata=None,
                 "`pip install pyAFQ[afqbrowser]` or "
                 "`pip install AFQ-Browser>=0.3`"))
             return
+        n_points_profile = self.export(
+            "n_points_profile", collapse=False)[
```

Comment:
Looks like this export call isn't working: https://github.com/yeatmanlab/pyAFQ/actions/runs/6880271899/job/18714094705?pr=1052#step:5:2321